test/testreference/atm_jjob_lgetkf_observer.ref

```diff
@@ -96,21 +96,21 @@ f10m                                         | Min:+9.2995810057091766e-01 Max:+
 ----------------------------------------------------------------------------------------------------
 H(x) for member 1:
 
-AMSUA N19 nobs= 83277 Min=201.8068522604288, Max=282.1977354171773, RMS=233.8559301511745
+AMSUA N19 nobs= 83277 Min=201.8068522615118, Max=282.197735417631, RMS=233.8559301511771
 
-sondes nobs= 2812 Min=-33.46923510113491, Max=237.4511515234925, RMS=66.49450460686083
+sondes nobs= 8481 Min=-33.72958083245494, Max=309.9582199847107, RMS=141.2113994268784
```

Comment:
Related to the concerns about changes in the references. This has to be departures, not H(x) right? How could we have a H(x) of temperature be -33K?

Reply:
Perhaps it includes all variables in the min/max/rms calculation. Because the max looks like a reasonable max for temperature while the min looks like a reasonable min for winds.

Reply:
Good point. I bet that's what it is. Makes these diagnostics super useful to include all variables! :-)